sdk/storage/Azure.Storage.Blobs.ChangeFeed/src/ChangeFeedFactory.cs

```diff
@@ -39,10 +40,11 @@ internal class ChangeFeedFactory
         }
 
         public async Task<ChangeFeed> BuildChangeFeed(
```

Comment:
`BuildChangeFeedInternal`

Reply:
Looks public in this, but is on an internal class.

Reply:
I think we should be fine - [Public things on an internal class are still internal](https://stackoverflow.com/questions/9302236/why-use-a-public-method-in-an-internal-class).